### DISCUSSION TITLE
Abus SHLM10010 supported?

### DISCUSSION THREAD

**Discussion by TheoW11** [zwave-js/zwave-js-ui#3512] [Q&A]
Hi,

I have lookup this device in the [data base browser](https://devices.zwave-js.io/?jumpTo=0x0403:0x0003:0x0002:0.0), the content seems to be empty.
Is the Abus SHLM10010 ([Led bulb 9W](https://mobil.abus.com/int/Consumer/Smart-Home/Z-Wave-extensions/Sensors-and-actuators/ABUS-Z-Wave-Light-Bulb))  supported icm with Domoticz?

Thanks,
Theo.

**Comment by kpine** [comment]
Yes, you just won't see any configuration parameters. The [config file](https://zwave-js.github.io/node-zwave-js/#/config-files/overview) would need to be updated to add them.

**Reply by TheoW11** [reply]
Will it still be possible to associate a motion sensor with the led bulb?

**Reply by kpine** [reply]
You may be thinking about that backwards. If you want to turn on the bulb when the motion sensor is triggered, you would assign the light bulb to the motion sensor's group(s). So that all depends on the support of the motion sensor, you would need to check its support for association groups. Any groups provided by the bulb aren't relevant.

According to the [manual](https://www.abus.com/var/ImagesPIM/d110001/medias/docus/22/ABUS_SHLM100x0_BDA_EN_1.3.pdf) I found, no, you cannot associate anything to the light bulb except the controller. It only has support for a) the lifeline group 1 and b) 1 device in that group.
